.github/workflows/ci-cd-docker.yml

```diff
@@ -0,0 +1,66 @@
+name: Build and push a new docker image based on project release tag.
+on:
+  release:
+    types: [published]
```

Comment:
this workflow should run also on pushing on the development branch

Reply:
done.